internal/pkg/utils/url/url.go

```diff
@@ -0,0 +1,44 @@
+package url
+
+import (
+	"net/url"
+	"strings"
+
+	"github.com/keboola/go-utils/pkg/orderedmap"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+)
+
+// ParseQuery is taken from net/url package but returns ordered map instead of regular map.
+// The implementation does not support array values.
+func ParseQuery(query string) (m *orderedmap.OrderedMap, err error) {
```

Comment:
No a při parsování form/post requestu potřebuju zachovat pořadí, tak jsem okopíroval `parseQuery` z `net/url` a namísto `map` použil `OrderedMap`. Na parametry typu pole jsem se vyprdl, tady je potřebovat (doufám 🙂) nebudeme.

Reply:
No to vím jak napsat 🙂 ale my v tom POST requestu podporujeme pole? Co to bude znamenat, když přijde? V jednom requestu by měl být jeden row, ne?

Reply:
no znamena to to iste ako ked niekto posle json `{"myarray": ["foo", "bar"]}`, proste to ako JSON ulozime do `body` stlpca. Vo form-data je validne posielat polia - nejaky kluc viac krat.

Reply:
Po implementacii `jsonnet` si nad tym array mozes potom definovat nejake vlastne operacie.

Reply:
Priklad form data array: https://community.postman.com/t/sending-an-array-as-form-data/4606/1

Reply:
Upravil jsem teda takto: https://github.com/keboola/keboola-as-code/pull/1008/commits/3990f6585c8f85f7b0ac68275d65f82133688697

(Ale query tohodle typu `list[0][id]=1&list[0][name]=one&list[1][id]=2&list[1][name]=two` to nezmerguje dohromady, nebo bych to měl řešit taky?)

Reply:
bude to v tý orderedmap takto: 
![CleanShot 2022-11-23 at 13 02 17](https://user-images.githubusercontent.com/215660/203542133-8ad27a29-5dc9-41fa-86a2-73792e739611.jpg)

Reply:
Nemusis, mozet zalozit task do cooldown, ze to este zvazime, ono to bude par riadkov naviac.